bin/ref_arch_setup

```diff
@@ -33,37 +37,124 @@ BANNER
   end
 end
 
-subcommands = {
+def install_global_opts(opt)
+  opt.on("--user value", "username for bolt to use to connect to target hosts") do |value|
+    @options["user"] = value
+  end
+  opt.on("--password value", "passworf for bolt to use to connect to target hosts") do |value|
+    @options["password"] = value
+  end
+  opt.on("--private-key value", "private key for bolt to use to connect to target hosts") do |value|
+    @options["private_key"] = value
+  end
+end
+
+def install_generate_opts(opt)
+  opt.on("--primary-master value", "hostname of primary master") do |value|
```

Comment:
Should we indicate required parameters, perhaps by either replacing "value" with "required" or "optional", or starting the description with "(required)"?